### PR TITLE
restore: add load stats description

### PR DESCRIPTION
**Pull request by 3pointer**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by Leavrth — APPROVED** [review]

File: br/br-snapshot-manual.md [hunk 1]
```diff
@@ -124,7 +124,7 @@ br backup full \
 --ignore-stats=false
 ```
 
-完成上述配置后，在恢复数据时，如果备份中包含了统计信息，br 命令行工具在恢复数据时会自动恢复备份的统计信息：
+完成上述配置后，在恢复数据时，如果备份中包含了统计信息，br 命令行工具在恢复数据时会自动恢复备份的统计信息（在 8.0 版本以后，新增 `--load-stats` 来控制是否恢复统计信息，默认恢复，一般情况下不需要关闭）：
```
Comment (qiancai) [inline br/br-snapshot-manual.md:127]:
```suggestion
完成上述配置后，在恢复数据时，如果备份中包含了统计信息，br 命令行工具在恢复数据时会自动恢复备份的统计信息（从 v8.0.0 起，BR 新增 `--load-stats` 参数来控制是否恢复备份的统计信息，默认恢复，一般情况下不需要关闭）：
```

File: br/br-snapshot-manual.md [hunk 2]
```diff
@@ -124,7 +124,7 @@ br backup full \
 --ignore-stats=false
 ```
```
Comment (qiancai) [inline br/br-snapshot-manual.md:126]: L115 这里的“和恢复”是不是可以删除
image

**Review by qiancai — APPROVED** [review]

File: br/br-snapshot-manual.md [hunk 3]
```diff
@@ -124,7 +124,7 @@ br backup full \
...
 
-完成上述配置后，在恢复数据时，如果备份中包含了统计信息，br 命令行工具在恢复数据时会自动恢复备份的统计信息：
+完成上述配置后，在恢复数据时，如果备份中包含了统计信息，br 命令行工具在恢复数据时会自动恢复备份的统计信息（从 v8.0.0 起，BR 新增 `--load-stats` 参数来控制是否恢复备份的统计信息，默认恢复，一般情况下不需要关闭）：
```
Comment (qiancai) [inline br/br-snapshot-manual.md:127]:
```suggestion
完成上述配置后，在恢复数据时，如果备份中包含了统计信息，br 命令行工具在恢复数据时会自动恢复备份的统计信息（从 v8.0.0 起，BR 新增 `--load-stats` 参数来控制是否恢复备份的统计信息，默认恢复，一般情况下无需关闭）：
```

File: br/br-snapshot-manual.md [hunk 4]
```diff
@@ -124,7 +124,7 @@ br backup full \
 --ignore-stats=false
 ```
 
-完成上述配置后，在恢复数据时，如果备份中包含了统计信息，br 命令行工具在恢复数据时会自动恢复备份的统计信息：
+完成上述配置后，在恢复数据时，如果备份中包含了统计信息，br 命令行工具在恢复数据时会自动恢复备份的统计信息（从 v8.0.0 起，BR 新增 `--load-stats` 参数来控制是否恢复备份的统计信息，默认恢复，一般情况下无需关闭）：
```
Comment (qiancai) [inline br/br-snapshot-manual.md:127]:
```suggestion
完成上述配置后，在恢复数据时，如果备份中包含了统计信息，br 命令行工具在恢复数据时会自动恢复备份的统计信息（从 v8.0.0 起，br 命令行工具新增 `--load-stats` 参数来控制是否恢复备份的统计信息，默认恢复，一般情况下无需关闭）：
```

**Review by lilin90 — APPROVED** [review]
/lgtm

File: br/br-snapshot-guide.md [hunk 5]
```diff
@@ -217,6 +217,13 @@ TiDB 备份功能对集群性能（事务延迟和 QPS）有一定的影响，
     --log-file restorefull.log
     ```
 
+- BR 通过参数 `--tikv-max-restore-concurrency` 来控制作业队列的最大长度，从而减少内存的消耗。作业队列的最大长度与 TiKV 数量成正比，与参数 `--tikv-max-restore-concurrency` 成正比，比例系数为 32。当硬盘吞吐接近网络带宽时，会导致低优先级的 ingest 任务堆积，从而导致作业队列堆积在 ingest 任务上，此时，我们需要设置参数 `--ratelimit` 来限制下载速度，让 ingest 任务有足够的资源。可以参考以下示例：
+
++ 当任意 TiKV 节点硬盘吞吐量为 x MB/s，任意 TiKV 下载备份文件的网络带宽大于 x/2 MB/s，可以设置参数 `--ratelimit x/2`。
++ 当任意 TiKV 节点硬盘吞吐量为 x MB/s，任意 TiKV 下载备份文件的网络带宽小于或等于 x/2 MB/s，可以不设置参数 `--ratelimit`。
+
+- 另外，我们也可以通过增大参数 `--tikv-max-restore-concurrency` 来增加作业队列的最大长度。当我们在监控面板 `TiKV-Details / Backup & Import / Import RPC count` 中发现 download 指标长时间接近于 0，而 ingest 一直处于限制上限时，说明 ingest 任务仍然堆积了，并且打到作业队列的最大长度，可以调高参数 `--tikv-max-restore-concurrency` 来增加作业队列的最大长度。
+
```
Comment (qiancai) [inline br/br-snapshot-guide.md:226]:
```suggestion
```

Comment (qiancai) [reply br/br-snapshot-guide.md:226]: can we move it to https://github.com/pingcap/docs-cn/pull/16740/files?

**Review by qiancai — APPROVED** [review]

**Pull request merged**